### PR TITLE
Create Bookshelf component

### PR DESCRIPTION
**Pull request by vladi-stanchev**

### CODE REVIEW

File: src/App.js
Comment (bertrowett) [inline src/App.js:1]: Remove semi-colons throughout, as they're not necessary for functionality here. You can fight us if you disagree!

File: src/Components/BookCard/bookcard.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.bookcard {
+    /* width: 20vw; */
+    /* padding: 20px;
+    margin: 20px; */
```
Comment (bertrowett) [inline src/Components/BookCard/bookcard.css:4]: delete if no longer necessary?

File: src/App.js
Comment (vladi-stanchev) [reply src/App.js:1]: Challenge accepted!

File: src/Components/BookCard/bookcard.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    justify-content: center;
+    align-items: center;
+    width: 20vw;
+    border-radius: 20px;
+    padding: 2rem 0;
+}
+
+.bookcard img {
+    max-width: 15vw;
+    /* margin-top: 2rem; */
```
Comment (bertrowett) [inline src/Components/BookCard/bookcard.css:17]: remove?

Comment (vladi-stanchev) [reply src/Components/BookCard/bookcard.css:17]: done

**Review by bertrowett — APPROVED** [review]

**Pull request merged**